HomeUI/src/views/apps/Management.vue

```diff
@@ -4507,8 +4504,8 @@ export default {
         const countryCode = locations[1];
         const regionName = locations[2];
         this.forbiddenGeolocations[`selectedContinent${i}`] = continentCode;
-        this.forbiddenGeolocations[`selectedCountry${i}`] = countryCode || 'NONE';
-        this.forbiddenGeolocations[`selectedRegion${i}`] = regionName || 'NONE';
+        this.forbiddenGeolocations[`selectedCountry${i}`] = countryCode || 'ALL';
```

Comment:
I think this was ok, on forbidden, if none is selected on continent it should say None not all. As it will not set anything if the it's on the default choice.